test/functional/lua/treesitter_spec.lua

```diff
@@ -16,37 +16,33 @@ before_each(clear)
 describe('treesitter API', function()
   -- error tests not requiring a parser library
   it('handles missing language', function()
-    eq('Error executing lua: .../treesitter.lua: no such language: borklang',
+    eq("Error executing lua: .../treesitter.lua: couldn't find tree-sitter parser for 'borklang' language",
```

Comment:
We should spell it "treesitter" everywhere that we control. Although it's probably not necessary in a error message, just "parser" is enough. We are unlikely to have other parser engines (or if that day ever comes we can rewrite the messages...).

Reply:
we could even just `no parser for 'borklang'`